packages/apps-config/src/ui/colors.ts

```diff
@@ -47,6 +47,7 @@ const chainUniarts = '#b39ef7';
 const chainIdavoll = '#ff43ff';
 const chainSubDAO = 'linear-gradient(50deg, #F20092 0%, #FF4D5D 100%)';
 const chainTrustBase = '#ff43aa';
+const chainSgc = 'linear-gradient(45deg, #0099F7 0%, #F11712 100%)';
```

Comment:
Same here ...

I have been cleaning up as I go through, there is still a bit that is not in the correct order depending on the file.

Reply:
OK the latest change is third add sgc network Thank you very much!!!